LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -351,7 +351,12 @@ DWORD ReadFromPipe(LPVOID Param)
                 }
             }
             else {
-                chBufOut[outSz++] = *ptr;
+                // no support for multibyte characters for now
+                // TODO(nandaa): https://github.com/microsoft/windows-container-tools/issues/121
+                if (*ptr < 0) {
+                    chBufOut[outSz++] = '?';

```

Comment:
Why are we adding the ? character to child process out buffer?

Reply:
tagged you in our chat conversation on Teams. This is just a way to show that we can't render multi-byte characters for now. It's an open bug -> #121